### PR TITLE
CC63590: Formatting content.

### PR DESCRIPTION
**Pull request by v-maudel**
Hello, @mairaw ,
This proposed file change comes from https://github.com/dotnet/docs.pl-pl/pull/157.
Could you review this contribution and help to merge if agreed?

Many thanks in advance.

### CODE REVIEW

**Review by mairaw — APPROVED** [review]

**Comment by mairaw** [comment]
Build stuck

**Comment by v-maudel** [comment]
Thanks @mairaw

File: docs/visual-basic/language-reference/statements/end-keyword-statement.md [hunk 1]
```diff
@@ -8,86 +8,91 @@ helpviewer_keywords:
...
 
 |Part|Description|
 |---|---|
 |`#End`|Required. Terminates the definition of the preprocessing block.|
-|`#ExternalSource`|Required to terminate an external source block begun by a matching [#ExternalSource Directive](../../../visual-basic/language-reference/directives/externalsource-directive.md).|
-|`#If`|Required to terminate a conditional compilation block begun by a matching `#If` directive. See [#If...Then...#Else Directives](../../../visual-basic/language-reference/directives/if-then-else-directives.md).|
-|`#Region`|Required to terminate a source region block begun by a matching [#Region Directive](../../../visual-basic/language-reference/directives/region-directive.md).|
+|`#ExternalSource`|Required to terminate an external source block begun by a matching [#ExternalSource Directive](../directives/externalsource-directive.md).|
+|`#If`|Required to terminate a conditional compilation block begun by a matching `#If` directive. See [#If...Then...#Else Directives](../directives/if-then-else-directives.md).|
+|`#Region`|Required to terminate a source region block begun by a matching [#Region Directive](../directives/region-directive.md).|
```
Comment (warchlak) [inline docs/visual-basic/language-reference/statements/end-keyword-statement.md]: This is second part of #End statement. Should be without `#`

Comment (mairaw) [reply docs/visual-basic/language-reference/statements/end-keyword-statement.md]: Thank you @warchlak. I've created a new PR to take care of this: #6434

**Pull request merged**